README.md

```diff
@@ -8,22 +8,77 @@
   <br>
 </h1>
 
-# Features
-- hot reloading
-- basic functionality
+This module implements the [`rdk:component:sensor` API](https://docs.viam.com/components/sensor) and provides two sensor models:
 
-# Models
 ```
 zaporter:sds011:v1
 zaporter:sds011:v1-fake
 ```
-# Example Config
+
+The `zaporter:sds011:v1` model supports the SDS011 Nova PM air quality sensor.
+The `zaporter:sds011:v1-fake` model can be used for testing the module without hardware.
+
+[!NOTE]
+> For more information, see [Modular Resources](https://docs.viam.com/registry/#modular-resources).
```

Comment:
Could we move this link to be on the `[module]` in `This module implements the [`rdk:component:sensor` API](https://docs.viam.com/components/sensor) and provides two sensor models:`

?

The NOTE sections feel a bit too close together / it makes it fairly cluttered

Reply:
That's a great idea. I was just going based on the template that we have in the docs https://docs.viam.com/registry/create/#optional-create-a-readme but you have a number of great pieces of feedback that I think we might want to apply to other module readmes as well (i.e. change the template). I'm going to make a ticket for the docs team to review all your feedback--thanks!